packages/graphql/tests/integration/directives/fulltext.int.test.ts

```diff
@@ -74,672 +94,2415 @@ describe("@fulltext directive", () => {
+                `Field "${invalidField}" is not defined by type`
+            );
+        });
+        test("Sorting by score ascending", async () => {
+            const query = `
+                query {
+                    ${queryType}(phrase: "a different name", sort: { score: ASC }) {
+                        score
+                        ${personTypeLowerFirst} {
+                            name
+                        } 
+                    }
+                }
+            `;
+            const gqlResult = await graphql({
+                schema: generatedSchema,
+                source: query,
+                contextValue: {
+                    driver,
+                    driverConfig: { database: databaseName },
+                },
+            });
+
+            expect(gqlResult.errors).toBeFalsy();
+            expect((gqlResult.data?.[queryType] as any[])[0][personTypeLowerFirst].name).toBe(person3.name);
+            expect((gqlResult.data?.[queryType] as any[])[1][personTypeLowerFirst].name).toBe(person1.name);
+            expect((gqlResult.data?.[queryType] as any[])[2][personTypeLowerFirst].name).toBe(person2.name);
+            expect((gqlResult.data?.[queryType] as any[])[0][SCORE_FIELD]).toBeLessThanOrEqual(
+                (gqlResult.data?.[queryType] as any[])[1][SCORE_FIELD]
+            );
+            expect((gqlResult.data?.[queryType] as any[])[1][SCORE_FIELD]).toBeLessThanOrEqual(
+                (gqlResult.data?.[queryType] as any[])[2][SCORE_FIELD]
+            );
+        });
+        test("Sorting by node", async () => {
+            const query = `
+                query {
+                    ${queryType}(phrase: "a different name", sort: [{ ${personTypeLowerFirst}: { name: ASC } }]) {
+                        score
+                        ${personTypeLowerFirst} {
+                            name
+                        } 
+                    }
+                }
+            `;
+            const gqlResult = await graphql({
+                schema: generatedSchema,
+                source: query,
+                contextValue: {
+                    driver,
+                    driverConfig: { database: databaseName },
+                },
+            });
+
+            expect(gqlResult.errors).toBeFalsy();
+            expect((gqlResult.data?.[queryType] as any[])[0][personTypeLowerFirst].name).toBe(person3.name);
+            expect((gqlResult.data?.[queryType] as any[])[1][personTypeLowerFirst].name).toBe(person2.name);
+            expect((gqlResult.data?.[queryType] as any[])[2][personTypeLowerFirst].name).toBe(person1.name);
+            expect((gqlResult.data?.[queryType] as any[])[0][SCORE_FIELD]).toBeNumber();
+            expect((gqlResult.data?.[queryType] as any[])[1][SCORE_FIELD]).toBeNumber();
+            expect((gqlResult.data?.[queryType] as any[])[2][SCORE_FIELD]).toBeNumber();
+        });
+        test("Unordered sorting", async () => {
```

Comment:
I'm unsure what this is trying to test, given that sort arguments are provided?! The `born` sort should be applied first, and then the `name` sort.

Reply:
Yeah I wrote I test for both ordered/unordered sorting e.g. (`sort: { score: ASC }` vs `sort: [{ score: ASC }]`. I did actually run into errors on only these tests when working on this PR so I think they should definitely be kept

Reply:
Good catch, super thorough testing!